r/R/duckdb.R

```diff
@@ -29,7 +29,7 @@
 #' @param table_name a name to use in DuckDB for this object. The default is a
 #' unique string `"arrow_"` followed by numbers.
 #' @param auto_disconnect should the table be automatically cleaned up when the
-#' resulting object is removed (and garbage collected)? Default: `TRUE`
+#' resulting object is removed (and garbage collected)? Default: `FALSE`
```

Comment:
Can you add a note explaining the consequences of cleaning up, or choosing not to clean up? As a user, I see that there is an option, but I have no idea why I would want to change the value or not.

Reply:
I added it to the description so the argument isn't super super long, but I can move it if you think it's better in the argument documentation block itself